Flow.Launcher/Languages/en.xaml

```diff
@@ -231,4 +232,40 @@
     <system:String x:Key="update_flowlauncher_update_files">Update files</system:String>
     <system:String x:Key="update_flowlauncher_update_upadte_description">Update description</system:String>
 
+    <!--  Welcome Window  -->
+    <system:String x:Key="Skip">Skip</system:String>
+    <system:String x:Key="Welcome_Page1_Title">Welcome to Flow Launcher</system:String>
+    <system:String x:Key="Welcome_Page1_Text01">Hello, It's your first time to run a Flow Launcher!</system:String>
+    <system:String x:Key="Welcome_Page1_Text02">Before starting, proceed with a simple setting. Surely, You can skip. Please choose the language you use.</system:String>
+    <system:String x:Key="Welcome_Page2_Title">Search and run everything in everywhere</system:String>
+    <system:String x:Key="Welcome_Page2_Text01">Search everything from applications, files, bookmarks, YouTube, Twitter and more. All from the comfort of your keyboard without ever touching the mouse.</system:String>
+    <system:String x:Key="Welcome_Page2_Text02">Flow Launcher starts with the hotkey below, start with the hotkey below. To change, select input and press the wanted hotkey in keyboard.</system:String>
+    <system:String x:Key="Welcome_Page3_Title">Hotkeys</system:String>
+    <system:String x:Key="Welcome_Page4_Title">Action Keyword and Commands</system:String>
+    <system:String x:Key="Welcome_Page4_Text01">Search the web or run various functions. Certain functions start with the expression action keyword, and if necessary, they can be used without action keywords. Try the queries below in Flow Launcher.</system:String>
```

Comment:
Search the web, launch applications or run various functions through Flow Launcher plugins. Certain functions start with an action keyword, and if necessary, they can be used without action keywords. Try the queries below in Flow Launcher.